core/src/test/scala/unit/kafka/admin/DeleteConsumerGroupsTest.scala

```diff
@@ -16,6 +16,7 @@
  */
 package unit.kafka.admin
 
+import joptsimple.OptionException
```

Comment:
With this import, `joptsimple.` can be dropped from the expected exception class of the first method below.
Also, no need for the `fail` statement there (similar to @ijuma's earlier comment). Not your code, but it would be great to clean them up :)

Reply:
Yes it makes sense. I have done the same for the `testConfigUsingNewConsumerUnrecognizedOption` in the `ConsumerPerformanceTest` class.

Reply:
Perfect. Thanks!